akka-sample-cluster-scala/src/main/scala/sample/cluster/stats/App.scala

```diff
@@ -0,0 +1,61 @@
+package sample.cluster.stats
+
+import akka.actor.typed.ActorSystem
+import akka.actor.typed.Behavior
+import akka.actor.typed.receptionist.Receptionist
+import akka.actor.typed.receptionist.ServiceKey
+import akka.actor.typed.scaladsl.Behaviors
+import akka.actor.typed.scaladsl.Routers
+import akka.cluster.typed.Cluster
+import com.typesafe.config.ConfigFactory
+
+object App {
+
+  val StatsServiceKey = ServiceKey[StatsService.ProcessText]("StatsService")
+
+  private object RootBehavior {
+    def apply(): Behavior[Nothing] = Behaviors.setup[Nothing] { ctx =>
+      val cluster = Cluster(ctx.system)
+      if (cluster.selfMember.hasRole("compute")) {
+        // on every compute node there is one service instance that delegates to N local workers
+        val numberOfWorkers = ctx.system.settings.config.getInt("stats-service.workers-per-node")
+        val workers = ctx.spawn(Routers.pool(numberOfWorkers)(StatsWorker()), "WorkerRouter")
```

Comment:
when the consistent hashing router is ready this could be a good use case for it, since StatsWorker has a cache per word, maybe create a ticket to remember to update?

Reply:
#145